build-scripts/hack/update-component-versions.py

```diff
@@ -59,6 +63,9 @@ def get_cni_version() -> str:
 
         raise Exception(f"Failed to find cni dependency in {deps_file}")
 
+def pull_contour_chart() -> None:
+    LOG.info("Pulling Contour Helm chart from %s with version %s", CONTOUR_HELM_REPO, CONTOUR_CHART_VERSION)
+    util.helm_pull("contour", CONTOUR_HELM_REPO, CONTOUR_CHART_VERSION, CHARTS)
```

Comment:
we seem to have more charts than that, as you have taken out some CRDs into a separate helm chart. This would have to be updated as well.


- please either document or write a script that updates these charts as well, as needed. for example, if we pull in a different `ck-contour`, we need to adjust `ck-contour-common` and `ck-contour-gateway` accordingly.
- please compress the big charts you have added, and add a version, e.g. `ck-contour-common-17.0.4-ck1.tgz`

Reply:
I chose a script and compressed the charts 👍🏼